themes/bootstrap5/templates/Helpers/date-range-slider.phtml

```diff
@@ -0,0 +1,25 @@
+<?php
+  $this->headScript()->appendFile('vendor/nouislider.min.js');
+  $this->headLink()->appendStylesheet('vendor/nouislider.min.css');
+  $this->headScript()->appendFile('date_range_slider.js');
+  $future = (int)date('Y', time() + 31536000); // next year
+  $low = !empty($this->values[0]) ? (int)$this->values[0] : 1400;
```

Comment:
Maybe worth introducing `$past = 1400;` since the value is reused in multiple places.